teams/js/home/dashboard.js

```diff
@@ -0,0 +1,3 @@
+function toggleSidebarButton() {
+    toggleSidebar();
```

Comment:
i put this in here to test js loading per-page, i know i can just put the function in the button directly but that doesn't test this